src/user/invite.js

```diff
@@ -50,7 +51,7 @@ module.exports = function (User) {
 
 		const data = await prepareInvitation(uid, email, groupsToJoin);
 
-		await emailer.sendToEmail('invitation', email, meta.config.defaultLang, data);
+		await emailer.sendToEmail('invitation', email, meta.config.defaultLang, data).catch(err => winston.error('[emailer.send] ' + err.stack));
```

Comment:
Here is another case. Not only were errors caught in `Emailer.send`, but also in `Emailer.sendViaFallback`. Removing the `sendViaFallback` one caused a test to fail, so I've added this catch to resolve that test failure.